components/TaskDetails.tsx

```
@@ -1,121 +1,138 @@
+      data-testid="task-details-1"
+      className="absolute top-0 left-0 mt-auto h-full w-full overflow-hidden rounded-none border border-gray-200 bg-white shadow-xs shadow-gray-400 md:static md:max-w-2xl md:rounded-xl"
+    >
+      <div className="space-y-6 p-6">
+        <div className="flex flex-row items-center justify-between">
+          <h2 className="text-xl font-semibold text-indigo-600">{title}</h2>
+          <div className="flex flex-row gap-2">
+            <button
+              onClick={onAcknowledge}
+              data-testid="details-acknowledge-button"
+              className="flex cursor-pointer flex-row items-center justify-center gap-2 rounded-md bg-indigo-600 px-4 py-2 text-sm font-medium text-white hover:bg-indigo-700 focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2 focus:outline-hidden"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)